docs/sources/http_api/folder_dashboard_search.md

```diff
@@ -97,3 +97,17 @@ Content-Type: application/json
   }
 ]
 ```
+## cURL examples
+
+The most basic example for a dashboard for which there is no authentication required is:
+
+```curl
```

Comment:
I don't think it's correct to use curl as language specifier for code blocks, since it's not a language. The language here would be a form of shell script, f.ex. bash. See [GitHub reference](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting).
```suggestion
```
```